src/effects/VST3/VST3Utils.cpp

```diff
@@ -55,7 +57,8 @@ namespace
 void FillParameters(
    wxWindow* parent,
    Steinberg::Vst::IEditController* editController,
-   Steinberg::Vst::IComponentHandler* handler)
+   Steinberg::Vst::IComponentHandler* handler,
+   const VST3Wrapper& wrapper)
```

Comment:
> The second commit fails to compile for me, unless I add to VST3Utils.h:
> 
> ```
> #include <vector>
> ```

Apologies, it was not yet clear to me that I should cmake-generate with pch off, when doing development.  When turning it off, I actually found that I'd also need `#include <functional>`

Reply:
> This commit adds an argument to FillParameters and BuildPlainUI. Instead, this could replace the editController argument, and that can be fetched from the wrapper.

good point, will do that